proc-macros/src/render_client.rs

```diff
@@ -210,6 +210,8 @@ impl RpcDescription {
 		param_kind: &ParamKind,
 		signature: &syn::TraitItemFn,
 	) -> TokenStream2 {
+		const ILLEGAL_PARAM_NAME: &str = "__________________params";
```

Comment:
The only wee thing is that this might provoke some sort of clippy warning about _ being used in front of a variable that's in use, but perhaps not an issue (and you could call it eg jsonrpsee__params or whatever if it was) :)